cpp/include/legate_dataframe/core/column.hpp

```diff
@@ -82,6 +82,31 @@ class LogicalColumn {
     }
   }
 
+  /**
+   * @brief Create a column with a legate array as the data
```

Comment:
This duplicates the cudf data type version, which I expect to remove in the future.